README.md

```diff
@@ -0,0 +1,32 @@
+# Khiops Documentation Website
+This is the technical repository for the Khiops documentation website.
+
+## Setup
+Install the website requirements
+
+```bash
+pip install -r requirements.txt
+```
+
+Then, to run locally just execute
```

Comment:
Add mention that we need to set-up the environment variables for the release versions of khiops, khiops python, viz, coviz, in order for the templates to get correctly instantiated?

Reply:
That is not part of the setup.